### PR TITLE
Make samples objects consistent across languages

### PR DESCRIPTION
**Pull request by simonz130**
- Changed go and node samples to comply with the same pattern for object naming.
Example: *go-hello-world* instead of hello-world.
This will help to use the same template for CI across all the languages without IF statements that check for special cases (like it is today)

### CODE REVIEW

File: golang/go-guestbook/kubernetes-manifests/guestbook-mongodb.deployment.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 kind: Deployment
 apiVersion: apps/v1
 metadata:
-  name: guestbook-mongodb
+  name: go=guestbook-mongodb
```
Comment (quoctruong) [inline golang/go-guestbook/kubernetes-manifests/guestbook-mongodb.deployment.yaml]: I think this should be `go-` instead of `go=`?

Comment (simonz130) [reply golang/go-guestbook/kubernetes-manifests/guestbook-mongodb.deployment.yaml]: Thanks, for the catch!

**Comment by simonz130** [comment]
Finally, all the builds are passing with the new triggers setup.

@quoctruong, could you please sign off?

**Pull request merged**